### ISSUE TITLE
[FatimaFawaz] deepcognition.ai soda bottle competition (10h)

### ISSUE THREAD

**Issue by shadiakiki1986** [teamshadi/ssoc18#5]
Hi @FatimaFawaz

We already started building a model in deepcognition.ai competition (#2).
The model is the same one that was demo'd for their "toilets" dataset.
The most recent training achieved 14% accuracy on a validation dataset, which is not much better than chance (since the target is a classification with 8 options, random selection has a 12.5% accuracy).
Please help us improve our model with the following steps:

- [x] warm up (~5h)
    - [x] play around with deepcognition.ai
    - [x] replicate the tutorial [MNIST Handwritten Digit Classification (using DNN)](https://deepcognition.ai/resources/videos/#mnist-dnn)
        - achieved 75% accuracy with 10 epochs
- [x] model version 1 (~5h)
    - [x] run the same model for MNIST above but on their "toilet dataset" (available under "public datasets")
        - 2018-07-02 achieved 50% validation accuracy with only 2 epochs so far due to memory / ETA problems
        - 2018-07-03 achieved 58% validation accuracy
        - will try to get better results
    - [x] run the same model on the soda bottle competition dataset
        - 2018-07-03 got 71% validation accuracy
    - [x] make a submission
        - got score = 25%
- [ ] ~~~model version 2: add a pre-trained layer such as VGG~~~ (0h)
    - this is to benefit from transfer learning ([wikipedia](https://en.wikipedia.org/wiki/Transfer_learning))
    - example deepcognition.ai usage of VGG can be found [here](https://community.deepcognition.ai/t/training-not-starting-with-pre-trained-model-with-include-top-layers-true/169)
    - Accuracy here should be better than random chance (12.5%) and better than version 0 of our model (14%)
    - More references:
        - [machinelearningmastery.com](https://machinelearningmastery.com/transfer-learning-for-deep-learning/)
        - [learnopencv.com: Transfer Learning using pre-trained models in Keras](https://www.learnopencv.com/keras-tutorial-transfer-learning-using-pre-trained-models/)
        - [learnopencv.com: Keras Tutorial : Fine-tuning using pre-trained models](https://www.learnopencv.com/keras-tutorial-fine-tuning-using-pre-trained-models/)
    - validation accuracy = ______
    - [ ] make a submission

**Comment by shadiakiki1986** [comment]
Hi @FatimaFawaz
Can you please write here a summary of the steps that you've done on the deepcognition.ai soda bottle competition so far? Also, please export your models as python keras code and upload them into a folder on the ssoc18.teamshadi.net server so we can commit it to the repository

**Comment by FatimaFawaz** [comment]
So far:

- I've made the same model on the Soda Bottle Competition dataset
- Made a submission
- Added a pre-trained VGG layer
- Got a higher training accuracy but no validation accuracy (trained on 10 epochs)
- Did not submit it

**Comment by shadiakiki1986** [comment]
let's try to 
- calculate a validation accuracy on your new model (with VGG)
- make a submission to compare it to the previous 25%

**Comment by FatimaFawaz** [comment]
I sent a message regarding this on Monday, achieved 32% accuracy with VGG.

**Comment by shadiakiki1986** [comment]
Yes I closed the issue because the competition itself is over too
